core/commands/round/create/create_round_command_handler.py

```diff
@@ -18,26 +16,15 @@ def __init__(self):
         self.golfcourse_repository = GolfcourseRepository(Golfcourse)
 
     def handle(self, command: CreateRoundCommand) -> Result[RoundDto]:
-
-        serializer = CreateRoundCommandSerializer(
-            data={
-                'golfcourseid': command.golfcourseid,
-                'dateplayed': command.dateplayed
-            })
-
-        if not serializer.is_valid():
-            return Result.fail(serializer.errors, status_code=400)
-
         if not self.golfcourse_repository.golfcourse_exists(golfcourseid=command.golfcourseid):
```

Comment:
This round_exists should be obsolete when we define the roundid REFERENCE as a PrimaryRelatedKeyField() -- just like the golfcourse PR open atm